src/LiquidationPair.sol

```diff
@@ -100,6 +103,14 @@ contract LiquidationPair {
       uint256(_virtualReserveIn) * _virtualReserveOut >= _minK,
       "LiquidationPair/virtual-reserves-greater-than-min-k"
     );
+    require(
+      UFixed32x4.unwrap(_maxPriceImpactOut) < 1e4,
+      "LiquidationPair/max-price-impact-greater-than-100"
```

Comment:
I guess that makes sense too. The price impact is a percentage, so the `100` in the error is referring to "100%" rather than the unformatted number of `_maxPriceImpactOut`.